[FIX]:Modify the configuration of ODP mode parameters (#85)
oceanbase/flink-connector-oceanbase: docs/sink/flink-connector-obkv-hbase.md

}
```

##### odp mode

```java
package com.oceanbase;

import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
import org.apache.flink.table.api.EnvironmentSettings;
import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;

public class Main {
    public static void main(String[] args) throws Exception {
        StreamExecutionEnvironment env =
                StreamExecutionEnvironment.getExecutionEnvironment().setParallelism(1);
        StreamTableEnvironment tEnv =
                StreamTableEnvironment.create(
                        env, EnvironmentSettings.newInstance().inStreamingMode().build());

        tEnv.executeSql(
                "CREATE TABLE t_sink ( "
                        + " rowkey STRING,"
                        + " family1 ROW<column1 STRING, column2 STRING>,"
                        + " PRIMARY KEY (rowkey) NOT ENFORCED"
                        + ") with ("
                        + "  'connector'='obkv-hbase',"
                        + "  'odp-mode'='true',"
                        + "  'odp-ip'='127.0.0.1',"

>                         + "  'odp-port'='12881',"

Reviewer: It's better to take the default rpc '2885' port for example.

Author: updated 2885

## Configuration

|          Option          | Required | Default |   Type   |                                                       Description                                                        |
|--------------------------|----------|---------|----------|--------------------------------------------------------------------------------------------------------------------------|

> | url                      | Yes      |         | String   | The config url, can be queried by <code>SHOW PARAMETERS LIKE 'obconfig_url'</code>.Required when using obconfig_url mode |

Reviewer: ```suggestion
| url                      | No      |         | String   | The config url, can be queried by <code>SHOW PARAMETERS LIKE 'obconfig_url'</code>. Required when 'odp-mode' is set to 'false'. |
```

| schema-name              | Yes      |         | String   | The database name of OceanBase.                                                                                          |
| table-name               | Yes      |         | String   | The table name of HBase, note that the table name in OceanBase is <code>hbase_table_name$family_name</code>.             |
| username                 | Yes      |         | String   | The username of non-sys tenant.                                                                                          |
| password                 | Yes      |         | String   | The password of non-sys tenant.                                                                                          |

> | odp-mode                 | No       | false   | Boolean  | If the value of ODP-MODE is true, use the ODP mode to connect to OBKV.                                                   |

Reviewer: ```suggestion
| odp-mode                 | No       | false   | Boolean  | If set to 'true', the connector will connect to OBKV via ODP.                                                 |
```

> | odp-ip                   | No       |         | String   | IP address of the ODP. When the value of odp-mode is true, it is required                                                |

Reviewer: ```suggestion
| odp-ip                   | No       |         | String   | IP address of the ODP. Required if 'odp-mode' is set to 'true'.        |
```

> | odp-port                 | No       |         | Integer  | rpc_listen_port of ODP. When the value of odp-mode is true, it is required.                                              |

Reviewer: ```suggestion
| odp-port                 | No       |         | Integer  | rpc_listen_port of ODP. Required if 'odp-mode' is set to 'true'.                                             |
```

> | sys.username             | No       |         | String   | The username of sys tenant.Required when using obconfig_url mode                                                         |

Reviewer: ```suggestion
| sys.username             | No       |         | String   | The username of sys tenant. Required if 'odp-mode' is set to 'false'.                                                          |
```

> | sys.password             | No       |         | String   | The password of sys tenant.Required when using obconfig_url mode                                                         |

Reviewer: ```suggestion
| sys.password             | No       |         | String   | The password of sys tenant. Required if 'odp-mode' is set to 'false'.                                                        |
```
